### ISSUE TITLE
How to access the aspect tooltip

### ISSUE THREAD

**Issue by andandio** [AstroDraw/AstroChart#11]
Hovering on an aspect line renders a default tooltip with the text "Test text". I've looked through all the possible settings in `settings.js` and searched the repo for the tooltip insertion point, but I don't see a way to customize this. Is there one?

**Comment by afucher** [comment]
Hey @andandio , I did not found this tooltip here. 
Were did you saw this tooltip? It shows in our examples?

**Comment by andandio** [comment]
@afucher , the example links in the read.me are suddenly showing up as plain text files. I thought they were live examples before?

I see where the tooltip is coming from now. I looked through every file in Chrome sources tab and finally found it:
<img width="1164" alt="Screen Shot 2021-11-06 at 4 15 35 PM" src="https://user-images.githubusercontent.com/7750668/140626621-8e467744-e5ac-44e4-88af-47372dfde7fa.png">
<img width="704" alt="Screen Shot 2021-11-06 at 4 16 09 PM" src="https://user-images.githubusercontent.com/7750668/140626625-0e1d204e-d121-48f3-bff0-1678f1dcb749.png">

I installed AstoChart as this npm package: [astrochart-modified](https://npm.io/package/astrochart-modified), which I see now points to the old (Kibo) repo and is several minor versions behind this one. So, I guess the tooltip is coming from that older version. Sorry for the confusion. I will add this repo to my `package.json` instead.
